microsoft-edge/webview2/release-notes/index.md

```diff
@@ -63,6 +63,351 @@ No additional APIs have been promoted to Stable and added in this Release SDK.
+* `FrameworkElement` -> `HwndHost` -> `WebView2`.
+* `FrameworkElement` -> `Control` -> `WebView2CompositionControl`.
+
+Background: If you're building a Windows Presentation Foundation (WPF) app and using the WebView2 control, you may find that your app runs into "airspace" issues, where the WebView2 control is always displayed on top, hiding any WPF elements in the same location, even if you try to specify the WPF elements to be above the WebView2 control (using visual tree order or the z-index property, for example).
+
+This issue occurs because the WPF control uses the WPF `HwndHost` to host the Win32 WebView2 control, and `HwndHost` has an issue with airspace; see [Mitigating Airspace Issues In WPF Applications](https://dwayneneed.github.io/wpf/2013/02/26/mitigating-airspace-issues-in-wpf-applications.html).
+
+See also:
+* [PR 4804: WPF Airspace - WebView2CompositionControl](https://github.com/MicrosoftEdge/WebView2Feedback/pull/4804/files?short_path=ebbc3ee#diff-ebbc3ee3560606e2823d68c134ea4aebdc1cb1252aaa9aa2b9a2815e2d8d36b2) - Spec.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* `CoreWebView2CompositionController` Class
+   * [CoreWebView2CompositionController.DragStarting Event](/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller.dragstarting?view=webview2-dotnet-1.0.3079-prerelease&preserve-view=true)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2 Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.3079-prerelease&preserve-view=true)<!-- todo: delete link to large old class -->
+<!-- todo: add new members? -->
+
+* `CoreWebView2CompositionController` Class
+   * [CoreWebView2CompositionController.DragStarting Event](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2compositioncontroller?view=webview2-winrt-1.0.3079-prerelease&preserve-view=true#dragstarting)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ExperimentalCompositionController6](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalcompositioncontroller6?view=webview2-1.0.3079-prerelease&preserve-view=true)
+   * [ICoreWebView2ExperimentalCompositionController6::add_dragstarting](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalcompositioncontroller6?view=webview2-1.0.3079-prerelease&preserve-view=true#add_dragstarting)
+   * [ICoreWebView2ExperimentalCompositionController6::remove_dragstarting](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalcompositioncontroller6?view=webview2-1.0.3079-prerelease&preserve-view=true#remove_dragstarting)
+
+---
+
+
+<!-- ---------- -->
+###### Load a WebView2 without a white flash (DefaultBackgroundColor API)
+
+Prevents a white flash during loading a WebView2.
```

Comment:
```suggestion
The `DefaultBackgroundColor` API allows users to set the `DefaultBackgroundColor` property at initialization. This prevents a disruptive white flash during the WebView2 loading process.
```

Reply:
result:
```
The DefaultBackgroundColor API allows users to set the `DefaultBackgroundColor` property at 
initialization.  This prevents a disruptive white flash during the WebView2 loading process.
```
i think our convention is no backtick on name of broad API (spec-level name).